### PR TITLE
Merge 1.7.7.x into develop - 06/03/2020

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merge last changes from 1.7.7.x into develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18012)

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+on: [push, pull_request]
+jobs:
+  php-cs-fixer:
+    runs-on: ubuntu-latest
+    name: PHP Coding Standards Fixer
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.3'
```
Comment (matks) [inline .github/workflows/php.yml:11]: ❤️

**Comment by PierreRambaud** [comment]
Travis is dying!

**Comment by matks** [comment]
## Integration tests failing on Travis because of
```
* Installing PrestaShop, this may take a while ...
file_get_contents_curl failed to download
http://www.prestashop.com/ajax/controller.php?email=demo%40prestashop.com&method=addMemberToNewsletter&language=0&visitorType=1&source=installer :
(error code 28) Operation timed out after 5674 milliseconds with 0 bytes receivedInstalled failed, displaying errors from logs:
```

1) `http` instead of `https`
2) Travis does not need to be tracked, let's skip this test env
3) even in prod, this should not prevent the shop from being installed

## Sanity tests failing on Travis because of
```
* Running puppeteer sanity tests ...
> puppeteer@1.0.0 sanity-travis /home/travis/build/PrestaShop/PrestaShop/tests/puppeteer
> ./node_modules/mocha/bin/mocha --bail campaigns/sanity/*
  Install Prestashop
    1) should open the Install page
  0 passing (31s)
  1 failing
  1) Install Prestashop
       should open the Install page:
     TimeoutError: Navigation Timeout Exceeded: 30000ms exceeded
      at Promise.then (node_modules/puppeteer/lib/LifecycleWatcher.js:142:21)
```

**Comment by matks** [comment]
Thank you @PierreRambaud

**Pull request merged**